new-readme.md

```diff
@@ -7,7 +7,7 @@ A JavaScript framework for **CQRS**, **Domain-Driven Design** and **Event Sourci
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-technologies.png"></p>
 
-* The [Installation](#installation) is fully automated and takes 2 minutes.
+* The [Installation](#installation) is fully automated and takes just<!-- sic --> 2 minutes.
```

Comment:
=> 2 minutes installation
I have never seen the npm packages without automated installation

Reply:
2 minutes is very long, I doubt it's an advantage. Those who get used to `apt` and `pip` (me) are amazed with the size and slowness of `npm`